package/kedro_viz/data_access/repositories/tracking_datasets.py

```diff
@@ -28,15 +21,9 @@ def __init__(self):
         ] = defaultdict(list)
 
     def get_tracking_datasets_by_group_by_run_ids(
-        self, run_ids: List[str], group: Optional[TrackingDatasetGroup]
+        self, run_ids: List[str], group: TrackingDatasetGroup
```

Comment:
Let's change `group: TrackingDatasetGroup = None` in `def run_tracking_data` to remove the `= None` default value too (assuming it's never called without the `group` argument anywhere on the frontend, which I'm pretty sure it's not).